files/common/usr/lib/systemd/scripts/nfs-version-init.sh

```diff
@@ -14,7 +14,7 @@
 #
 versions=$(cat /proc/fs/nfsd/versions)
 if [[ "$versions" == "-2 -3 -4 -4.0 -4.1 -4.2" ]]; then
-	printf "+4\n" >/proc/fs/nfsd/versions
+	printf "+4\\n" >/proc/fs/nfsd/versions
```

Comment:
Why is this being escaped?  The kernel looks for this return char.

Reply:
It comes down to the same, it's just that shellcheck complains:
```
$ printf "+4\\n" > /tmp/1
$ printf "+4\n" > /tmp/2
$ md5sum /tmp/1 /tmp/2
4a61639514b388ae3d602fe6852aec76  /tmp/1
4a61639514b388ae3d602fe6852aec76  /tmp/2
```